Source/Events.Store.MongoDB/Events/EventMetadata.cs

```diff
@@ -32,8 +32,11 @@ public EventMetadata(DateTimeOffset occurred, Guid eventSource, Guid typeId, uin
 
         /// <summary>
         /// Gets or sets the <see cref="DateTimeOffset"/> of when the event was committed to the event store.
+        /// BsonType.Document saves a UTC DateTime, ticks and an offset(in minutes) to the document. This way we can
+        /// query for the DateTime from the database and it looks nicer than the string representation.
+        /// https://github.com/mongodb/mongo-csharp-driver/blob/master/src/MongoDB.Bson/Serialization/Serializers/DateTimeOffsetSerializer.cs#L158 .
```

Comment:
This can be inside a <remarks></remarks> tags

Reply:
added